roles/rsyslog/tasks/inputs/files/main.yml

```diff
@@ -9,12 +9,12 @@
 
 - name: Create files input configuration files in /etc/rsyslog.d
   vars:
-    __rsyslog_input_decl: 'input(type="imfile" file="{{ files_item.rsyslog_input_log_path | d(rsyslog_input_log_path) }}" tag="{{ files_item.name }}")'
     __rsyslog_input_name: "{{ files_item.name }}"
     __rsyslog_input_type: "{{ files_item.type }}"
     __rsyslog_packages: []
     __rsyslog_rules: [ { "name": "input-files-{{ files_item.name }}", "type": "input",
-                         "sections": [ { "options": "{{ lookup('template', 'input_template.j2') }}" } ] } ]
+                         "sections": [ { "options":
+                                           "input(type=\"imfile\" file=\"{{ files_item.rsyslog_input_log_path | d(rsyslog_input_log_path) }}\" tag=\"{{ files_item.name }}\")\n {{ lookup('template', 'input_template.j2') }}" } ] } ]
```

Comment:
I must say that the value of the variable becomes longer and uglier, with an embedded template and quoting, this does not look like a step in the right direction.

Reply:
Yeah, I was hoping to share one template input_template.j2 among all the input types... Do you think I should give it up?

Reply:
reformat the variable to make it more YAML-ish, and move the entire `options` value into the template:
```yaml
     __rsyslog_rules:
       - name: "input-files-{{ files_item.name }}"
         type: input
         sections:
           - options: "{{ lookup('template', 'input_template.j2') }}"
```
You'll also have to change the template to use `input_item` instead of `files_item`

Reply:
thank you @richm , I have had something like this on mind, if it is feasible.

Reply:
Than you for your suggestions, @richm, @pcahyna!
Could you please review this commit? (Since input_template.j2 does not refer `files_item` directly, the changes are only needed in the tasks, I think.)
https://github.com/linux-system-roles/logging/pull/113/commits/990c66d32b1a4e52402fc0aa28e95bf5075d1cd5